### ISSUE TITLE
Insufficient color contrast on table data

### ISSUE THREAD

**Issue by awkawk** [w3c/wai-axsdb-web#45]
In test results the style for cells containing "Not tested" the color contrast is 2.13:1 and needs to be improved to meet the 4.5:1 ratio required by WCAG 2.0.
